metal/classifier.py

```diff
@@ -142,10 +144,27 @@ def _train(self, train_loader, loss_fn, X_dev=None, Y_dev=None):
                 model_class, **checkpoint_config, verbose=self.config["verbose"]
             )
 
+        # Moving model to GPU
+        if train_config["use_cuda"]:
+
+            if self.config["verbose"]:
+                print("Using GPU...")
+
+            self.cuda()
+
         # Train the model
         for epoch in range(train_config["n_epochs"]):
             epoch_loss = 0.0
-            for data in train_loader:
+            for batch, data in tqdm(
```

Comment:
'batch' can be a lot of things--e.g., data, or a counter. In this case, it's a counter, right? So maybe, batch_num.